react/week2/todoapp/src/todo-app.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import Timer from './Timer'
+
+function TodosList({ id, description, removeTodo }) {
+    const [isDone, setIsDone] = useState('');
+    return (
+        <div>
+            <li className="list-of-todos" style={isDone ? { textDecoration: "line-through" } : { textDecoration: "" }}>{description}
+                <span>
+                    <input type="checkbox" onClick={() => setIsDone(!isDone)} />
+                    <button onClick={() => removeTodo(id)}>Delete</button>
+                </span>
+            </li>
+        </div>
+    )
+}
+
+const RenderedTodos = ({ todos }) => {
+    const [addTodo, setAddTodo] = useState(todos);
+    const [value, setValue] = useState('');
+    //adding the todos to todos array
+    const OnClickAddTodo = () => {
+        if (value !== "") {
+            const myNextTodos = { id: addTodo.length + 1, description: value };
+            const myNextTodoNames = addTodo.concat(myNextTodos);
+            setAddTodo(myNextTodoNames)
+        }
+    }
+    //Remove the todos
+    const removeTodo = (removeId) => {
+        const newTodos = addTodo.filter((data) => {
+            return data.id !== removeId
+        })
```

Comment:
This is correct but just make it simple.
addTodo.filter(data=>data.id !== removeId)
when u have more than one line code than you can use like u did otherwise make it clean :)